book/snippets/nostr/python/src/nip65.py

```diff
@@ -0,0 +1,24 @@
+from nostr_protocol import EventBuilder, Tag, Kind, Keys, RelayMetadata
+
+def nip65():
+    # Get Keys
+    keys = Keys.generate()
+
+    print()
+    print("Relay Metadata:")
+    # ANCHOR: relay-metadata
+    # Create relay metadata tags
+    tag1 = Tag.relay_metadata("wss://relay.damus.io", RelayMetadata.READ)
+    tag2 = Tag.relay_metadata("wss://relay.primal.net", RelayMetadata.WRITE)
+    tag3 = Tag.relay_metadata("wss://relay.nostr.band", None)
+
+    # Build/sign event
+    kind = Kind(10002)
+    content = ""
+    tags = [tag1,tag2,tag3]
+    builder = EventBuilder(kind,content,tags)
+    event = builder.to_event(keys)
```

Comment:
There is an easier way to construct event kind 10002, by using `EventBuilder.relay_list`. You have to pass just a dictionary where the key is the relay url and the value the metadata (read, write or none).

Reply:
Ah, that would be a better option. Will get this fixed probably Monday, away from comp for a day or two.

Reply:
Updated, I included both methodologies. However, please let me know if you prefer we only include the one.